services/core/IEEE_2030_5/README.md

```diff
@@ -0,0 +1,211 @@
+# VOLTTRON 2030.5 Agent #
+
+The VOLTTRON 2030.5 agent facilitates communication between the PlatformDriverAgent and a 2030.5
+server, using the IEEE 2030.5(2018) protocol. The agent's primary role is to manage data exchange
+between the PlatformDriverAgent and the 2030.5 server. It creates MirrorUsagePoints and readings on
+the 2030.5 server and dispatches control actions to the message bus during DERControl events.
+
+The following diagram illustrates the data flow for the 2030.5 agent, from the PlatformDriverAgent to the 2030.5 server:
+
+```mermaid
+  sequenceDiagram
+    Agent->>Server: Creates MirrorUsagePoints
+    Server-->>Client: 201 OK
+    Agent->>PlatformDriverAgent: Subscribes to Device Data
+    PlatformDriverAgent->>Agent: Agent Receives Device Data
+    Agent->>Server: Posts MeterReadings(Device Data)
+    Agent->>PlatformDriverAgent: Publishes Event Controls
+```
+
+To see this process in action, please try out the [Agent Demo](AGENTDEMO.md).
+
+## Agent Config File ##
+
```

Comment:
I think we should document all three configuration files here and point out how they are related

Reply:
Something to the effect- 
Since 2030.5 agent works with a PlatformDriver agent to interface between a device and 2030.5 server, there are three configuration files involved - 
1. device configuration 
2. registry configuration files 
3.  2030.5 agent configuration. 
The first two configurations are used by the platform driver to interact with the device. The third,  2030.5 agent configuration file contains configurations related to both the server and a point mapping that maps volttron point names of a device to the DER point details.  The volttron point names in this point mapping should match with the volttron point name in the registry configuration file used the platform driver agent